modules/reserved-ips/variables.tf

```diff
@@ -0,0 +1,126 @@
+##############################################################################
+# VPC Variables
+##############################################################################
+
+variable "prefix" {
+  description = "The prefix that you would like to append to your resources"
```

Comment:
We should probably update this description to make it clear that the prefix is only used if no value is passed for the `vpe_name` option in the `cloud_services` input variable. We need to make sure its clear that they can use that option instead of prefix for full control over naming. I guess same change is needed in root level variable description too